bin/test-lxd-images

```diff
@@ -189,6 +189,14 @@ for url in $(lxc query "/1.0/instances" | jq -r .[]); do
         continue
     fi
 
+    # Wait for instance to be ready to execute commands
+    for i in $(seq 30); do
+        if lxc exec "${name}" -- true 2>/dev/null; then
+            break
+        fi
+        sleep 2
+    done
```

Comment:
How is that needed when we've already gone through a `sleep 1m` after the last instance has been started?

We also wait 10s after every 3 instance starts, so for large sets of instances (like our usual 100+), this means we'll have waited well over 6min by the time we run the tests here.